src/main/java/ua/edu/internship/interview/service/business/InterviewService.java

```diff
@@ -0,0 +1,152 @@
+package ua.edu.internship.interview.service.business;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ua.edu.internship.interview.data.documents.InterviewDocument;
+import ua.edu.internship.interview.data.documents.InterviewQuestionDocument;
+import ua.edu.internship.interview.data.documents.UserQuestionDocument;
+import ua.edu.internship.interview.data.enumeration.InterviewStatus;
+import ua.edu.internship.interview.data.repository.InterviewRepository;
+import ua.edu.internship.interview.data.repository.UserQuestionRepository;
+import ua.edu.internship.interview.service.dto.interview.InterviewCreateDto;
+import ua.edu.internship.interview.service.dto.interview.InterviewDto;
+import ua.edu.internship.interview.service.dto.interview.InterviewUpdateDto;
+import ua.edu.internship.interview.service.dto.interview.question.InterviewQuestionCreateDto;
+import ua.edu.internship.interview.service.dto.interview.question.InterviewQuestionDto;
+import ua.edu.internship.interview.service.dto.interview.question.InterviewQuestionUpdateDto;
+import ua.edu.internship.interview.service.mapper.InterviewMapper;
+import ua.edu.internship.interview.service.mapper.InterviewQuestionMapper;
+import ua.edu.internship.interview.service.utils.exceptions.InterviewCollisionException;
+import ua.edu.internship.interview.service.utils.exceptions.NoSuchEntityException;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class InterviewService {
+    private final InterviewRepository interviewRepository;
+    private final UserQuestionRepository userQuestionRepository;
+    private final InterviewMapper interviewMapper;
+    private final InterviewQuestionMapper questionMapper;
+
+    public List<InterviewDto> getAllInterviews() {
+        List<InterviewDocument> interviews = interviewRepository.findAll();
+        log.info("Retrieved {} interviews from database.", interviews.size());
+        return interviewMapper.toDto(interviews);
+    }
+
+    public InterviewDto getInterviewById(String id) {
+        InterviewDocument interview = getInterviewByIdOrElseThrow(id);
+        log.info("Retrieved interview with id: {}", id);
+        return interviewMapper.toDto(interview);
+    }
+//    TODO Validate available time? (For now lets implement it with validation:
+//     both interviewer and candidate available +-1h before and after interview)
+    public InterviewDto createInterview(InterviewCreateDto dto) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyLineSeparatorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'METHOD_DEF' should be separated from previous line.
<!-- __reviewdog__:ChAyOWI4ZjZlY2VhNmU2ZTRlEgpjaGVja3N0eWxl -->